pkgs/development/python-modules/algopy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage
+, lib
+, fetchurl
+, numpy
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "algopy";
+  version = "0.5.7";
+
+  src = fetchurl {
```

Comment:
In both cases I figured it'd be better to be closer to the original code base, instead of having another instance in between. If this is preferred I can of course use `fetchPypi` instead.

I'm rather new to pull requests. Will the changes be applied if I force push to my fork?

Reply:
Changed both packages to `fetchPypi`

Reply:
The main advantage of `fetchPypi` is that it's a bit more convenient and has support for using mirrors if the main server is down.